Kronos-Shifts-Connector/Deployment/azuredeploy.json

```diff
@@ -729,7 +838,7 @@
                 "attributes": {
```

Comment:
Do these secrets really need a "location"? It's not in the documentation: https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/2018-02-14/vaults/secrets

Reply:
The secrets do not need a location, I have removed them in the latest commit